### DISCUSSION TITLE
How to localize

### DISCUSSION THREAD

**Discussion by cermiosi** [JohanFalt/Foundry_WoD20#442] [Q&A] [answered]
I love this system and want to create a german localization for it.
I could do that by just editing the .html-s of the sheets. Yet i figured there has to be a more elegant way, but i dont understand from where calls like " localize 'wod.abilities.talents' " read their input.
Would be greatfull for any tips

**Comment by JohanFalt** [comment, marked as answer]
There are German already. What you need to do is set your Foundry to German under system settings. Restart foundry and you are good to go.
